modules/sap_hana/main.tf

```
@@ -14,15 +14,15 @@
  * limitations under the License.
  */
 
-terraform {
-  required_version = "~> 0.11.0"
-}
-
 module "sap_hana" {
   source        = "./sap_hana_python"
   instance-type = "${var.instance_type}"
```

Terraform 0.12 removed the need for most of the string interpolation required in previous versions.

Arguments which have values of single variables, resource attributes, or module outputs should be stripped of string interpolation:

```suggestion
  instance-type = var.instance_type
```

---

This still needs to be addressed.